services/cd-service/pkg/repository/transformer.go

```diff
@@ -806,24 +809,24 @@ func writeCommitData(ctx context.Context, h *db.DBHandler, transaction *sql.Tx,
 	return nil
 }
 
-func (c *CreateApplicationVersion) calculateVersion(ctx context.Context, transaction *sql.Tx, state *State) (uint64, error) {
+func (c *CreateApplicationVersion) calculateVersion(ctx context.Context, transaction *sql.Tx, state *State) (uint64, uint64, error) {
```

Comment:
it would make more sense to me if `calculateVersion` returns a `ReleaseNumbers` struct.